src/modules/import/persistence/import-vorgang.entity.ts

```diff
@@ -0,0 +1,34 @@
+import { TimestampedEntity } from '../../../persistence/timestamped.entity.js';
+import { Entity, Enum, Opt, Property } from '@mikro-orm/core';
+import { ImportStatus } from '../domain/import.enums.js';
+
+@Entity({ tableName: 'importvorgang' })
+export class ImportVorgangEntity extends TimestampedEntity {
+    @Property({ columnType: 'uuid', nullable: true })
+    public readonly importByPersonId?: string;
+
+    @Property()
+    public readonly importByUsername!: string;
+
+    @Property({ columnType: 'uuid', nullable: true })
+    public readonly rolleId?: string;
+
+    @Property()
+    public rollename!: string;
+
+    @Property({ columnType: 'uuid', nullable: true })
+    public readonly organisationId?: string;
```

Comment:
Yes, we talked about it in the refinement, because later it will have constiture a reference to the corresponding tables and we would like to keep the history of import even after the rolle or the organisation or the person is deleted.